modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-rollback_{context}"]
+= Rolling back the network provider to OpenShift SDN
+
+As a cluster administrator, you can rollback the Pod network provider for your cluster to the OpenShift SDN Pod network provider.
+During the rollback, you must reboot every node in your cluster.
+
+[IMPORTANT]
+====
+Only rollback to OpenShift SDN if the migration to the OVN-Kubernetes Pod network provider is unsuccessful.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare metal infrastructure configured with the OVN-Kubernetes Pod network provider.
+
+. To enable the migration, set an annotation on the Cluster Network Operator configuration object by entering the following command:
++
+----
+$ oc annotate Network.operator.openshift.io cluster \
+  'networkoperator.openshift.io/network-migration'=""
+----
+
+. To change the network provider, enter the following command:
++
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{ "spec": { "networkType": "OpenShiftSDN" } }'
+----
+
+. Optional: Use the backup of the cluster network configuration to restore any customizations to the network configuration that you might have made, such as the maximum transmission unit (MTU) value.
```

Comment:
Is there an `oc` command to restore the backup? How does the user do this?

Reply:
@mburke5678, it's really choose your own adventure; they user will need to manually restore customizations by running `oc edit Network.config`.

Reply:
@huiran0826, is it possible change MTU as part of a rollback? Normally it can be done only during cluster installation, I think.

Reply:
@mburke5678 perhaps:

> . Optional: Use the backup of the cluster network configuration that you created before the migration to restore any customizations to the network configuration that you might have made. To restore the customizations, enter the following command to edit the Cluster Network Operator configuration: